src/java/org/apache/cassandra/profiler/AsyncProfilerMBean.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.profiler;
+
+public interface AsyncProfilerMBean
+{
+    void start(String event, String outputFormat, int timeout, String outputPath);
```

Comment:
timeout - duration probably a better name for the parameter + a time unit as a a suffix would be helpful

outputPath - using a path is too dangerous from security point of view, a path to a data file or another file can be specified to cause a DoS (explicitly or using tricks like ../../.., so it is better to allow only a file name with explictly defined set of symbols to use (allowlist is safer than an denylist)

Reply:
I left it at timeout, as it is what the actual async profiler is using.

For the outputPath, that makes sense. Changing.